lib/test-utils.js

```diff
@@ -27,11 +27,15 @@ function createVertex(entity, tech) {
     if (typeof entity === 'string') {
         const p = entity.split('.');
 
-        entity = BemNaming.parse(p[0]);
+        entity = bemNaming.parse(p[0]);
         tech || (tech = p[1]);
     }
 
-    return new BemCell({ entity: new BemEntityName(entity), tech });
+    if (!BemEntityName.isBemEntityName(entity)) {
```

Comment:
I guess we can replace this part of code with `BemCell.create({ entity, tech })`